src/LasagnaController.cpp

```diff
@@ -106,7 +106,7 @@ void LasagnaController::innerLoop(float input_h){
    */
   float deadband_effort = 0;
   float thresh = constants.k_v*constants.k_h*constants.ss_error_thresh;
-  if(jankabs(state.effort)-thresh > 0){
+  if(pasta_abs(state.effort)-thresh > 0){
```

Comment:
This might be more readable as `if (pasta_abs(state.effort) < thresh)`

Reply:
I think u mean greater than but true